### DISCUSSION TITLE
v2版本的多主机管理功能是否需要多台专业版授权

### DISCUSSION THREAD

**Discussion by rdself** [1Panel-dev/1Panel#6794] [Q&A]
请问v2版本的多主机管理功能是否需要每台都有专业版授权？
还是说类似哪吒探针一样的主丛结构，只需要主机器有专业版授权，其他机器通过类似探针的程序进行控制，不再有单独的1panel面板？

**Comment by wanghe-fit2cloud** [comment]
目前多主机管理功能的具体授权策略还没有确定，我们正在集中精力研发 v2 版本的核心功能，至于授权的具体方案，将会在后续公布，感谢理解。
